api/src/services/permissions.ts

```diff
@@ -144,4 +145,89 @@ export class PermissionsService extends ItemsService {
 
 		return res;
 	}
+
+	async getItemPermissions(collection: string, primaryKey?: string): Promise<ItemPermissions> {
+		if (!this.accountability?.user) throw new ForbiddenError();
+
+		if (this.accountability?.admin)
+			return {
+				update: { access: true },
+				delete: { access: true },
+				share: { access: true },
+			};
```

Comment:
```suggestion
		if (this.accountability?.admin) {
			return {
				update: { access: true },
				delete: { access: true },
				share: { access: true },
			};
		}
```

As this is technically multi line we should use brackets for consistency.